src/common.jl

```diff
@@ -1,13 +1,17 @@
 ## Generic ##
 
-function vcatmapreduce(f, args...)
-    init = vcat(f(first.(args)...,))
-    zipped_args = zip(args...,)
-    return mapreduce(vcat, drop(zipped_args, 1); init = init) do zarg
-        f(zarg...,)
+_istracked(x) = false
+_istracked(x::AbstractArray{<:TrackedReal}) = true
```

Comment:
Should we also have a 
```julia
_istracked(x::TrackedArray) = true
```
?

Reply:
No need. This is only to see if we need to call `vcat` or not to return a `TrackedArray`.